### ISSUE TITLE
Traditional IOD protocol

### ISSUE THREAD

**Issue by zhongjian1999** [yaoyao-liu/CL-DETR#14]
Hi, I implemented this code successfully under the protocol used in CL-DETR, but the performance is very low when I try to train the first stage (1-40) under traditional IOD protocol. 
Additionally, I used the same random seed in this project (which is 42), so the class order is:
[74, 64, 65, 5, 79, 59, 36, 88, 9, 49, 10, 42, 86, 34, 82, 90, 72, 89, 50, 25, 2, 27, 75, 51, 67, 22, 16, 85, 18, 31, 21, 80, 7, 73, 57, 17, 81, 20, 15, 24, 11, 56, 13, 1, 6, 48, 4, 46, 14, 60, 39, 32, 35, 23, 38, 43, 58, 33, 40, 47, 8, 54, 70, 62, 87, 44, 3, 77, 61, 55, 78, 41, 28, 52, 37, 84, 53, 63, 19, 76],
where the classes in first stage (1-40) is [74, 64, ..., 15, 24,]. 
When I increase it to 1-44, which include [74, 64, ..., 15, 24, 11, 56, 13, 1], the performance is normal. After my several experiments, the 'person' (id: '1') is the reason leads to very low performance.
I suspect this is either because there are a large number of person instances or due to issues in my code implementation. I would appreciate it if you could help clarify my confusion or provide your trained weights.
Thanks in advance!

**Comment by yaoyao-liu** [comment]
Thanks for your interest in our project. The training process of the first stage is exactly the same as the original Deformable DETR. Because in the first stage, there are no continual learning techniques applied. The only difference is that we use different data. You may directly use [the original Deformable DETR code](https://github.com/fundamentalvision/Deformable-DETR) to train the model for the first stage. 

If you have any further questions, please feel free to let me know.

**Comment by zhongjian1999** [comment]
> Thanks for your interest in our project. The training process of the first stage is exactly the same as the original Deformable DETR. Because in the first stage, there are no continual learning techniques applied. The only difference is that we use different data. You may directly use [the original Deformable DETR code](https://github.com/fundamentalvision/Deformable-DETR) to train the model for the first stage.
> 
> If you have any further questions, please feel free to let me know.

Thank you for your reply. 
I re-examined the issue and found that a smaller learning rate is required for convergence when training classes 1-40 under the traditional protocol. 
Could you please provide the 1-40 weights or the 1-40 final performance  under the traditional protocol for my reference?
Thanks in advance!

**Comment by yaoyao-liu** [comment]
@zhongjian1999 

Thanks for sharing your implementation details. Currently, I don't have the checkpoints for the traditional IOD protocol on my current server. These files are saved on the server of my PhD institution. I will contact my previous colleagues to see if they can help me transfer the pre-trained checkpoints to my current server.

**Comment by zhongjian1999** [comment]
> [@zhongjian1999](https://github.com/zhongjian1999)
> 
> Thanks for sharing your implementation details. Currently, I don't have the checkpoints for the traditional IOD protocol on my current server. These files are saved on the server of my PhD institution. I will contact my previous colleagues to see if they can help me transfer the pre-trained checkpoints to my current server.

Hi, the results I obtained for 1-40 and 1-70 in the traditional protocol are 45.0 and 44.1, respectively. Are these close to your results at that time?
